x-pack/legacy/plugins/apm/server/lib/transactions/breakdown/index.ts

```diff
@@ -33,14 +80,14 @@ export async function getTransactionBreakdown({
           must: [
             {
               term: {
-                'service.name.keyword': {
+                [SERVICE_NAME]: {
                   value: serviceName
                 }
               }
             },
             {
               term: {
-                'transaction.type.keyword': {
+                'transaction.type': {
```

Comment:
constants for `'transaction.type'` and `'transaction.name'`

Reply:
changed all of the field names to constants from `common/elasticsearch_fieldnames.ts`